mumuki-laboratory.gemspec

```diff
@@ -34,5 +34,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'rack', '~> 2.0'
   s.add_dependency 'omniauth', '~> 1.4.0'
 
+  s.add_dependency 'will_paginate', '~> 3.1'
```

Comment:
Why do we need this? Isn't kaminari just enough?

Reply:
I didn't notice that we were using kaminari :stuck_out_tongue: